docs/source/networking/kaizen-networks.md

```diff
@@ -73,6 +73,9 @@ IP Address       Hostname/Description
 129.10.5.143     esi-controller-2.massopen.cloud
 129.10.5.144     rubicone-foreman.massopen.cloud
 129.10.5.145     esi-vip-external.massopen.cloud
+129.10.5.146     sso1.massopen.cloud (name resolves to CSAIL address)
+129.10.5.147     sso2.massopen.cloud (name resolves to CSAIL address)
+129.10.10.148    sso.massopen.cloud (VIP)
```

Comment:
I can't ping this address are you sure it is not 129.10.5.148?

Reply:
good catch! that was a typo. I fixed it now.